Blockzilla/UIComponents/URLBar.swift

```diff
@@ -298,7 +298,7 @@ class URLBar: UIView {
         }
         
         toolset.deleteButton.snp.makeConstraints { make in
-            make.trailing.equalTo(toolset.contextMenuButton.snp.leading).offset(isIPadRegularDimensions ? UIConstants.layout.deleteButtonOffset : 0)
+            make.trailing.equalTo(toolset.contextMenuButton.snp.leading).offset(isIPadRegularDimensions ? UIConstants.layout.deleteButtonOffset : -UIConstants.layout.deleteButtonMarginContextMenu)
```

Comment:
Looks like `deleteButtonOffset` has a negative value of -5 and `deleteButtonMarginContextMenu` is pozitive. In general we would like to keep them positive but `offset` seems to be orientation relative. I think we can use `inset` and keep the values positive.